plugins/user-authenticators/saml2/src/org/apache/cloudstack/saml/SAML2AuthManagerImpl.java

```diff
@@ -119,12 +123,12 @@
     @Override
     public boolean start() {
         if (isSAMLPluginEnabled()) {
-            setup();
             s_logger.info("SAML auth plugin loaded");
+            return setup();
         } else {
```

Comment:
we can remove else {} here just keep;
if (condition){
return stuff();
}
return otherstuff();